integration/test/ParseObjectTest.js

```diff
@@ -80,6 +80,14 @@ describe('Parse Object', () => {
     });
   });
 
+  it('can check if object exists', async () => {
+    const object = new TestObject();
```

Comment:
Could you also include a test here? I mean calling `await object.exists()` before saving for the first time.